Readme.md

```diff
@@ -17,26 +17,27 @@ OSRM Text Instructions has been translated into [several languages](https://gith
 
 ### JavaScript Usage
 
-```
+```js
 var version = 'v5';
-var language = 'en';
-var options = {};
-var osrmTextInstructions = require('osrm-text-instructions')(version, language, options);
+var options = {
+    languages: [ 'en', 'fr' ]
+};
+var osrmTextInstructions = require('osrm-text-instructions')(version, options);
 
-// make your request against the API
+// make your request against the API, save result to response variable
 
 response.legs.forEach(function(leg) {
   leg.steps.forEach(function(step) {
-    instruction = osrmTextInstructions.compile(step)
+    instruction = osrmTextInstructions.compile(language, step)
   });
 });
 ```
 
 parameter | required? | values | description
 ---|----|----|---
 `version` | required | `v5` | Major OSRM version
-`language` | required | `en` `de` `zh-Hans` `fr` `nl` `ru` | Language identifier
 `options.hooks.tokenizedInstruction` | optional | `function(instruction)` | A function to change the raw instruction string before tokens are replaced. Useful to inject custom markup for tokens
+`options.languages` | optional | `en` `de` `zh-Hans` `fr` `nl` `ru` | Array of language identifiers that should be supported. Default is loading all language files, which can be huge on websites
```

Comment:
We don’t want developers to get the impression that only these six languages are supported, given that we currently support ten ([with more to come](https://www.transifex.com/project-osrm/osrm-text-instructions/languages/)). Perhaps add “etc.” and link to [this folder](https://github.com/Project-OSRM/osrm-text-instructions/tree/master/languages/translations/) for the full list of supported languages?